src/main/java/com/example/bookshop/mapper/CartItemListMapper.java

```diff
@@ -0,0 +1,13 @@
+package com.example.bookshop.mapper;
+
+import com.example.bookshop.config.MapperConfig;
+import com.example.bookshop.dto.cartitem.CartItemDto;
+import com.example.bookshop.model.CartItem;
+import java.util.Set;
+import org.mapstruct.Mapper;
+
+@Mapper(config = MapperConfig.class, componentModel = "spring",

```

Comment:
I think you already had `componentModel = "spring"` in MapperConfig class. 
Please check other mappers